### DISCUSSION TITLE
Create triggers on the fly

### DISCUSSION THREAD

**Discussion by federico-terzi** [espanso/espanso#1207] [Feature requests and Ideas]
Suggested by @pwujek in #384, this is the proposal:

Would there be any possibility of being able to add simple triggers on the fly, as well as on the command line?

This would allow adding new triggers while typing in text, for example I used to interview people and as they were talking add Microsoft Word replacements as short forms for people's names and departments or acronyms that they referred to which allowed me to keep near verbatim records in real time.

For example entering something along the lines of :myTrigger='my replacement text', and as well as adding myTrigger to the config file it would do the replacement into the current text as well.

Also related to #288.

**Comment by smeech** [comment]
Python script [here](https://pastebin.com/96RApmkp).
